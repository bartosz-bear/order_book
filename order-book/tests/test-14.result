{"buyOrders": [{"id": 1, "price": 14, "quantity": 20}], "sellOrders": []}
{"buyOrders": [{"id": 2, "price": 16, "quantity": 3}, {"id": 1, "price": 14, "quantity": 20}], "sellOrders": []}
{"buyOrders": [{"id": 2, "price": 16, "quantity": 3}, {"id": 3, "price": 15, "quantity": 3}, {"id": 1, "price": 14, "quantity": 20}], "sellOrders": []}
{"buyOrders": [{"id": 3, "price": 15, "quantity": 1}, {"id": 1, "price": 14, "quantity": 20}], "sellOrders": []}
