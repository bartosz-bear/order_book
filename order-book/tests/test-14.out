{"buyOrders": [{"id": 1, "price": 14, "quantity": 20}], "sellOrders": []}
{"buyOrders": [{"id": 2, "price": 16, "quantity": 3}, {"id": 1, "price": 14, "quantity": 20}], "sellOrders": []}
{"buyOrders": [{"id": 2, "price": 16, "quantity": 3}, {"id": 3, "price": 15, "quantity": 3}, {"id": 1, "price": 14, "quantity": 20}], "sellOrders": []}
{"buyOrders": [{"id": 3, "price": 15, "quantity": 1}, {"id": 1, "price": 14, "quantity": 20}], "sellOrders": []}
{"buyOrderId": 2, "sellOrderId": 4, "price": 16,  "quantity": 3}
{"buyOrderId": 2, "sellOrderId": 4, "price": 16,  "quantity": 3}
{"buyOrderId": 2, "sellOrderId": 4, "price": 16,  "quantity": 3}
{"buyOrderId": 2, "sellOrderId": 4, "price": 16,  "quantity": 3}
{"buyOrderId": 2, "sellOrderId": 4, "price": 16,  "quantity": 3}
{"buyOrderId": 2, "sellOrderId": 4, "price": 16,  "quantity": 3}
{"buyOrderId": 2, "sellOrderId": 4, "price": 16,  "quantity": 2}
{"buyOrderId": 3, "sellOrderId": 4, "price": 15,  "quantity": 3}
{"buyOrderId": 3, "sellOrderId": 4, "price": 15,  "quantity": 3}
{"buyOrderId": 3, "sellOrderId": 4, "price": 15,  "quantity": 3}
{"buyOrderId": 3, "sellOrderId": 4, "price": 15,  "quantity": 3}
{"buyOrderId": 3, "sellOrderId": 4, "price": 15,  "quantity": 3}
{"buyOrderId": 3, "sellOrderId": 4, "price": 15,  "quantity": 3}
{"buyOrderId": 3, "sellOrderId": 4, "price": 15,  "quantity": 1}
